just_curious_game_test.exs

```diff
@@ -0,0 +1,97 @@
+defmodule JustCurious.GameTest do
+  use JustCurious.BotCase
+
+  setup do
+    no_chain_questions()
+    {:ok, %{game: standard_new_game()}}
+  end
+
```

Comment:
Note: I don't see a test for `break_tie` functionality! Seems like a critical path that should be accounted for.